features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
+        let dispatch = this.props.store.dispatch;
+        dispatch(localParticipantJoined());
+        dispatch(initLib(this.props.config));
+    }
+
+    /**
+     * Dispose lib-jitsi-meet and remove local participant when component is
+     * going to be unmounted.
+     *
+     * @inheritdoc
+     */
+    componentWillUnmount() {
+        let dispatch = this.props.store.dispatch;
+        dispatch(localParticipantLeft());
+        dispatch(disposeLib());
```

Comment:
My question above applies here as well. I guess the answer to the above question will answer the question here but, anyway, I'm asking here as well for the sake of completeness.